### DISCUSSION TITLE
Open the Snack.dashboard

### DISCUSSION THREAD

**Discussion by fdelacruz** [folke/snacks.nvim#127] [Q&A]
I have a keymap that simply opens the dashboard from any opened file. With now Snacks I tried Snacks.dashboard() but that looks ugly. Is there another way?

**Comment by folke** [comment]
update snacks if you mean with `ugly` that it opens in a floating window.
I changed the default yesterday to show fullscreen by default

**Reply by fdelacruz** [reply]
Thank you @folke
